.vscode/bevy.code-snippets

```diff
@@ -0,0 +1,75 @@
+{
+    "Bevy: New module-level function Plugin": {
+        "scope": "rust",
+        "prefix": "plugin_fn",
+        "body": [
+            "use bevy::prelude::*;",
+            "",
+            "pub(super) fn plugin(app: &mut App) {",
+            "\t$0",
+            "}"
+        ],
+        "description": "Define a new module-level function Plugin"
+    },
+    "Bevy: New Component": {
+        "scope": "rust",
+        "prefix": "component",
+        "body": [
+            "#[derive(Component, Reflect)]",
+            "#[reflect(Component)]",
+            "struct $1;"
+        ],
+        "description": "Define a new Component"
+    },
+    "Bevy: New Resource": {
+        "scope": "rust",
+        "prefix": "resource",
+        "body": [
+            "#[derive(Resource, Reflect, Default)]",
+            "#[reflect(Resource)]",
+            "struct $1;"
+        ],
+        "description": "Define a new Resource"
+    },
+    "Bevy: New Event": {
+        "scope": "rust",
+        "prefix": "event",
+        "body": [
+            "#[derive(Event)]",
+            "struct $1;"
+        ],
+        "description": "Define a new Event"
+    },
+    "Bevy: New SystemSet": {
+        "scope": "rust",
+        "prefix": "system_set",
+        "body": [
+            "#[derive(SystemSet, Copy, Clone, Eq, PartialEq, Hash, Debug)]",
+            "enum $1 {",
+            "\t$0",
+            "}"
+        ],
+        "description": "Define a new SystemSet"
+    },
+    "Bevy: New Schedule": {
+        "scope": "rust",
+        "prefix": "schedule",
+        "body": [
+            "#[derive(ScheduleLabel, Copy, Clone, Eq, PartialEq, Hash, Debug)]",
+            "struct $1;"
+        ],
+        "description": "Define a new Schedule"
+    },
+    "Bevy: New States": {
+        "scope": "rust",
+        "prefix": "states",
+        "body": [
+            "#[derive(States, Copy, Clone, Default, Eq, PartialEq, Hash, Debug, Reflect)]",
+            "enum $1 {",
+            "\t#[default]",
+            "\t$0",
+            "}"
+        ],
+        "description": "Define a new States"
```

Comment:
```suggestion
        "description": "Define new States"
```
I know why you wrote it the original way, but it reads weird

Reply:
It reads weird because `bevy_state` is weird ^\\\_(\`\`/)\_/^

Reply:
either change it to "Define new States" or "Define a new States enum", either are fine for me.

Reply:
I'm going to remove all of the descriptions because they don't provide any value over the names. It looks like this with no description:
![2024-08-09_1723241293_863x147](https://github.com/user-attachments/assets/1ccb35e1-48d7-493a-b6f9-466fe9435e8e)